code/src/robos_core.cpp

```diff
@@ -77,20 +77,45 @@ namespace r2d2::robos {
             // robos_core_c::robos role = moving_platform;
             break;
         case EXAMPLE_ROLE:
+            // auto moving_platform_role = robos_role_c();
+            // robos_core_c::robos role = moving_platform;
             break;
         }
+
         return 0;
     };
 
     int robos_core_c::run_role() {
+        const std::vector<frame_type> test_frame;
+        robos_core_c::current_role->run(test_frame);
```

Comment:
robos_core_c::current_role->run(test_frame); causes error
robos_core.cpp:(.text._ZN4r2d25robos12robos_core_c8run_roleEv+0x1a): undefined reference to `operator delete(void*)'
collect2.exe: error: ld returned 1 exit status